data/lis-nepal.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "LIS Nepal Pvt. Ltd.",
+    "establishedAt": 2014,
```

Comment:
The "establishedAt" field is a [UNIX Timestamp](https://www.unixtimestamp.com/) .